lib/paymentPage/paymentPage.dart

```diff
@@ -0,0 +1,336 @@
+import 'package:flutter/material.dart';
+import 'package:veganic_foods_app/constants.dart';
+import 'package:veganic_foods_app/landingPage/components/backgroundFunc.dart';
+import 'package:veganic_foods_app/paymentPage/components/background.dart';
+
+import 'components/CardPage.dart';
+import 'components/defaultAppBar.dart';
+import 'components/defaultBackButton.dart';
+
+enum Payment_methods { card, mtn, airtel, zanaco, fnb }
+
+class PaymentPage extends StatelessWidget {
+  const PaymentPage({Key? key}) : super(key: key);
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      children: [
+        Container(child: const BackGround()),
+        Container(
+          child: PaymentList(),
+        )
+      ],
+    );
+  }
+}
+
+class PaymentList extends StatefulWidget {
+  @override
+  State<PaymentList> createState() => _PaymentListState();
+}
+
+class _PaymentListState extends State<PaymentList> {
+  Payment_methods? _character = Payment_methods.card;
+  @override
+  Widget build(BuildContext context) {
+    Size size = MediaQuery.of(context).size; //total height and width of screen
+
+    return Scaffold(
+        backgroundColor: bGcolor,
```

Comment:
@batsy3 Fam i had already set up the default background and primary color. Kindly refer to the colors file in assets and reference the background color accordingly.